tests/test_endpoint_class.py

```diff
@@ -0,0 +1,39 @@
+import dxfeed as dx
+import pytest
+
+
+class ValueStorage(object):  # config
```

Comment:
Mb we can also add test that everything works fine `Endpoint` in case no arguments passed to it's constructor? (Default behaviour)